packages/terra-brand-footer/tests/wdio/brand-footer-spec.js

```diff
@@ -18,24 +18,77 @@ describe('BrandFooter', () => {
     });
   });
 
-  describe('Hovered', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-brand-footer/brand-footer/hovered-brand-footer');
+  describe('Links Prop', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-brand-footer/brand-footer/links-prop-brand-footer'));
+
+    Terra.should.beAccessible({ viewports, rules });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty({
+      '--terra-brand-footer-nav-background-color': 'pink',
+      '--terra-brand-footer-nav-color': 'yellow',
+      '--terra-brand-footer-link-color': 'green',
+      '--terra-brand-footer-footer-content-background-color': 'red',
+      '--terra-brand-footer-footer-content-color': 'black',
     });
+  });
 
+  describe('No Nav', () => {
+    beforeEach(() => browser.url('/#/raw/tests/terra-brand-footer/brand-footer/no-nav-brand-footer'));
+
+    Terra.should.beAccessible({ viewports, rules });
     Terra.should.matchScreenshot({ viewports });
     Terra.should.themeEachCustomProperty({
-      '--terra-brand-footer-link-hover-color': 'brown',
+      '--terra-brand-footer-footer-content-background-color': 'red',
+      '--terra-brand-footer-footer-content-color': 'black',
     });
   });
 
-  describe('Active', () => {
```

Comment:
How come you removed the hover and active tests?

Reply:
The tests weren't actually testing the active or hover styles, only applying a color variable that happens to be the same as it would be if the link was activated or hovered over. I don't think that merits testing.

I do now remember that there is actually a WDIO method to hover over an element, so that test will be added back.

Reply:
The moveToObject method would not activate the hover styling in my testing, so the hover test will be left off again.

Reply:
Did you wrap `moveToObject` in an `it` block?

Reply:
I did